generated/installer/ibm-spectrum-scale-csi-operator-dev.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       serviceAccountName: ibm-spectrum-scale-csi-operator
       containers:
       - name: operator
-        image: quay.io/ibm-spectrum-scale-dev/ibm-spectrum-scale-csi-operator:dev
```

Comment:
we should not change the images in dev.yaml file to cp.icr.io path

Reply:
reverted the changes in this file,